packages/ui/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 The following is a curated list of changes in the Enact ui module, newest changes on the top.
 
+## [unreleased]
+
+### Changed
+- `ui/VirtualList`, `ui/VirtualGridList`, and `ui/Scroller` components not to call `onScrollStop` too frequently during scroll
```

Comment:
`ui/Scroller` should be inclusive of `ui/VirtualList*`. Plus, it's missing the context for "non-animation" only case.

I suggest something like

> - non animating `ui/Scroller` to debounce `onScrollStop` events

Reply:
We normally write all components. I updated the description but still I hope someone changes it to more fluent description.